deep_qa/data/dataset_readers/squad_sentence_selection_reader.py

```diff
@@ -3,6 +3,8 @@
 import logging
 import os
 import random
+
+import nltk
```

Comment:
well having it in the `readFile` method actually doesnt do anything --- it still has to be imported always, since when pylint collects tests it imports everything from every test, and then only runs certain tests.